jest.preset.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    displayName: 'demosplan-ui',
+    testEnvironment: 'node',
```

Comment:
The mor I think about it, the more I get the feeling, that we can merge this file with the 'jest.config.js' and drop this line here since its wrong anyway

Reply:
done https://github.com/demos-europe/demosplan-ui/pull/51/commits/ceaad0f8c3da970d1aa46adb3975a1235d1dcbea and https://github.com/demos-europe/demosplan-ui/pull/51/commits/d5a1ae2e0e749c3be1cf0c98bcbc7fb42426834f